PRCS/prc-4.md

```diff
@@ -150,6 +150,117 @@ It's not an AMM. In a typical AMM dex, there are liquidity pools and you trade a
 
 The DEX contract on the base chain only facilitates the trading (transferring) of existing Inverse Projected ERC1155 assets, it does not make any assurances about validity of such assets. That aspect is handled by the feature set of the Inverse Projected ERC1155 standard itself. The responsibility of querying the API of game chain to check the validity of the sell order is left up to the front-end providers, as well as the presentation of the possible sell orders.
 
+## Game Node Dex API (OPTIONAL)
+
+These endpoints are provided by the game node to allow external sites generate a frontend for the DEX.
+
+1. Get Game Assets and Metadata.
+
+    `GET /dex/`
+
+    RESPONSE 
+    ```js
+    {
+        assets: {
+            code: string;          // Asset Code
+            description: string;   // Asset Description
+            fromSym: string;       // Name of base Asset  
+            toSym: string;         // Name of unit to convert
```

Comment:
Could we also get the contract address for the asset and for the asset DEX? (Note: each asset must have its own DEX).
And would be also good to have asset `name` (or is that what you mean by `description`?).
What I imagined:
```
{
code: 'tgold',
name: 'Tarochi Gold',
description: 'Main currency of the RPG game Tarochi',
fromSym: 'TGOLD',
toSym: 'ETH',
address: '0x...',
dexAddress: '0x...',
...
}
```